### PR TITLE
Fix the resource mismatch issue between template and scenarios

### PR DESCRIPTION
**Pull request by anpingli**
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/212717/console

### CODE REVIEW

**Comment by anpingli** [comment]
@QiaolingTang  PTAL

File: features/logging/cluster_logging_operator.feature [hunk 1]
```diff
@@ -126,13 +126,13 @@ Feature: cluster-logging-operator related test
     """
     And the expression should be true> elasticsearch('elasticsearch').policy_ref(name: 'app') == "app-policy"
     And the expression should be true> elasticsearch('elasticsearch').delete_min_age(name: "app-policy") == cluster_logging('instance').application_max_age
-    And the expression should be true> elasticsearch('elasticsearch').rollover_max_age(name: "app-policy") == "1h"
+    And the expression should be true> elasticsearch('elasticsearch').rollover_max_age(name: "app-policy") == "60m"
```
Comment (QiaolingTang) [inline features/logging/cluster_logging_operator.feature:129]: this value should be the `maxAge` in elasticsearch/elasticsearch. So here the values are `3m`,  `9m`, `1h`

**Comment by anpingli** [comment]
@QiaolingTang  The values are 3m, 9m, 1h now
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/212784/console

**Comment by QiaolingTang** [comment]
/lgtm
Thanks.

**Pull request merged**